### DISCUSSION TITLE
Issue with GitHub Pages

### DISCUSSION THREAD

**Discussion by mahdihasan2006** [imfing/hextra#175] [Q&A] [answered]
I have created two new repositories from [hextra-starter-template](https://github.com/imfing/hextra-starter-template). One is working and one is not. Links are given below. Can anyone check them and tell me what's the problem and solution?

1. Repo: https://github.com/LinuxBangla/LinuxBangla-Website
Website: https://linuxbangla.github.io/LinuxBangla-Website/

1. Repo: https://github.com/LinuxBangla/linuxbangla.github.io
Website: https://linuxbangla.github.io/

**Comment by imfing** [comment, marked as answer]
@mahdihasan2006 For your second site, could you change the `--baseURL` in the `.github/workflows/pages.yaml` following the guide below:

https://imfing.github.io/hextra/docs/guide/deploy-site/#github-pages
